src/main/java/creativei/service/impl/InquiryServiceImpl.java

```diff
@@ -100,4 +100,12 @@ public Inquiry update(Inquiry inquiry) throws  DataIntegrityException, InvalidPa
     public Inquiry updateAll(List<Inquiry> inquiries) {
         return null;
     }
+
+    @Override
+    public List<Inquiry> getUnattendedInquiry(String boolParam, String statusParam) {
+        Boolean isAttended;
+        isAttended=boolParam=="false"?false:true;
```

Comment:
Compare string using .equals method. Use ignore case

Reply:
Resolved in commit 26c4d3f